config/initializers/services.rb

```diff
@@ -0,0 +1 @@
+BOARDING_SERVICE = BoardingService.new
```

Comment:
Yeah, everything inside initializers is run during app startup, so by creating the service here, we get startup-time checking of all ENV values. That way we don't have to wait for page load to see if we forgot something